src/demoAskScriptServer.ts

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env node
```

Comment:
This should not be required to run this as script

Reply:
You added it to demoServer.ts, I just mimicked the same style.

![image](https://user-images.githubusercontent.com/12752520/85137474-93c4fb80-b241-11ea-9859-8eb39c199484.png)

Removing it from both files then.